tools/rosmsg/src/rosmsg/__init__.py

```diff
@@ -549,10 +549,14 @@ def iterate_packages(rospack, mode):
 def _get_package_paths(pkgname, rospack):
     paths = []
     path = rospack.get_path(pkgname)
+    path = os.path.normcase(os.path.normpath(path))
```

Comment:
As mentioned in another of your PRs the usage of `normpath` is problematic since if symlinks are present it might change the semantic of the path. Therefore please remove the usage of it.

Reply:
will remove `os.path.normpath`